### PR TITLE
Add cookies and analytics

### LINKED ISSUE #12: Cookie disclaimer

### PR DESCRIPTION
**Pull request by JakeSCahill**
Fixes #12

@Deflaimun I'm not sure how the optanon cookies work. How do we test it locally? I can't see anywhere in our current site where we inject a script for this cookie banner.

### CODE REVIEW

**Comment by Deflaimun** [comment]
I'm not sure if it's possible to test this locally. It's best to add Justin to this PR

**Comment by JakeSCahill** [comment]
Update: Reached out to Justin on Slack

**Comment by itsjustinc** [comment]
Hi all, sorry for the delay - Everything looks good with the exception of the Mouseflow script on line 8 of /head-scripts. This can be removed.

Mouseflow is injected by Google Tag Manager, so as long as you maintain the GTM container script, there's no need for a separate Mouseflow script.

File: src/partials/head-scripts.hbs [hunk 1]
```diff
@@ -3,6 +3,10 @@
 </script>
 <script>function gtag(){dataLayer.push(arguments)};window.dataLayer=window.dataLayer||[];gtag('js',new Date());gtag('config','{{this}}')</script>
 {{/with}}
+<meta name="google-site-verification" content="QcL-pD81oJatgKXQ3Wquvk_Ku3RRtUljxKoMaicySQA" />
+
+<script defer src="//cdn.mouseflow.com/projects/4260ac2a-9c53-42f1-a6cf-c2942fbfc263.js"></script>
```
Comment (JakeSCahill) [inline src/partials/head-scripts.hbs:8]:
```suggestion
```

**Review by Deflaimun — APPROVED** [review]

**Pull request merged**